test/common-utils/devcontainer-ruby-zshrc/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/devcontainers/ruby:3.2
+
+USER vscode
+RUN echo 'eval "$(rbenv init -)"' >> /home/vscode/.zshrc
```

Comment:
For better maintainability, it would be beneficial to structure it like the example below.

FROM mcr.microsoft.com/devcontainers/ruby:3.2

USER vscode
ENV USER=vscode
RUN echo 'eval "$(rbenv init -)"' >> /home/$USER/.zshrc

Reply:
done